hr_phone_validation/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Please follow the template of OCA: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst